### PR TITLE
Upgrade links to https and include Skylight

### PR DESCRIPTION
**Pull request by eliotsykes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -665,7 +665,8 @@ Commercial Products
...
 
 Rails has been lucky to have a few companies dedicated to Rails-specific
-performance tools. A couple of those are:
+performance tools:
 
-* [New Relic](http://www.newrelic.com)
-* [Scout](http://scoutapp.com)
+* [New Relic](https://www.newrelic.com)
+* [Scout](https://scoutapp.com)
+* [Skylight](https://www.skylight.io/)
```
Comment (JuanitoFatas) [inline README.md]: Could remove this `/` since above two url do not have, keep consistency. :blush:

Comment (eliotsykes) [inline README.md]: :+1:

**Comment by senny** [comment]
@eliotsykes can you please squash your commits?

**Comment by eliotsykes** [comment]
@senny Squashed :+1:

**Comment by senny** [comment]
@eliotsykes thank you :yellow_heart:

**Comment by eliotsykes** [comment]
You're welcome! @senny and thanks for the super speed feedback and PR merge!

**Pull request merged**